### PR TITLE
chore: (draft) add basic info to release notes for 3.2.

### PR DESCRIPTION
**Pull request by kgilmer**

### CODE REVIEW

File: content/docs/reference/Releases/regolith-3.2-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+## Features
+
+* Ubuntu 24.04 (Noble) support
+* `regolith-control-center` rebased from GNOME 46 (for supported disto releases)
+* Sway session promoted from `beta` to `stable` status
+* Overhaul of X11 and Wayland session initialization
+
+## Installation Instructions
+
+Follow the standard installation instructions, but substitute `testing` instead of `release-3_1`.  For example, this is the `apt` line for Ubuntu Mantic:
```
Comment (khos2ow) [inline content/docs/reference/Releases/regolith-3.2-release-notes.md:22]: Is this a typo? `Ubuntu Mantic`?

File: content/docs/reference/Releases/regolith-3.2-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+```shell
+echo deb "[arch=amd64 signed-by=/usr/share/keyrings/regolith-archive-keyring.gpg] \
+https://regolith-desktop.org/testing-ubuntu-noble-amd64 noble main" | \
```
Comment (khos2ow) [inline content/docs/reference/Releases/regolith-3.2-release-notes.md:26]: or this is supposed to be `mantic`?

File: content/docs/reference/Releases/regolith-3.2-release-notes.md (hunk 1 above)
Comment (kgilmer) [reply content/docs/reference/Releases/regolith-3.2-release-notes.md:22]: thx, yep I just missed it in search/replace

File: content/docs/reference/Releases/regolith-3.2-release-notes.md (hunk 2 above)
Comment (kgilmer) [reply content/docs/reference/Releases/regolith-3.2-release-notes.md:30]: fixed thanks

**Pull request merged**